scenarios/01_attack_of_the_orcs.cfg

```diff
@@ -238,17 +237,17 @@
 		[message]
 			speaker=Haldar
 			# po: "Ich habe einen Kobold entdeckt! Er steht auf meinem Feld!"
-			message= _ "I spotted a goblin! He stood on my field!"
+			message= _ "I spotted a goblin! He's standing on my field!"
 		[/message]
 		[message]
 			speaker=Ysedda
 			# po: "Ein Kobold... 'steht' auf deinem Feld? Du meinst, er hat dich angegriffen?"
-			message= _ "A goblin... 'stood' on your field? You mean, he attacked you?"
+			message= _ "A goblin... 'is standing' on your field? You mean, he attacked you?"
```

Comment:
Sicher?
Vielleicht steht er da ja gar nicht mehr. Kann Haldar in diesem Moment nicht wissen.

Reply:
Haldar denkt aber nicht so weit mit. Das ist durchaus beabsichtigt. Er sieht ein Problem, weiß nicht, wie er damit umgehen soll, also holt er sich Hilfe für dieses spezifische Problem.